lib/runner/variant-reassignment.js

```diff
@@ -266,6 +269,20 @@ export default class VariantReassignment {
     )
   }
 
+  /**
+   * In productDrafts from external systems, there's no productTypeId, but instead the name is used.
+   * However, productTypeId is needed for reassignment. This method replaces the productTypeName
+   * with productTypeId if such ID exists.
+   */
+  _resolveProductTypeReferences (productDrafts, productTypeIdToTypeObj) {
+    productDrafts.forEach((productDraft) => {
+      const productType = productTypeIdToTypeObj[productDraft.productType.id]
```

Comment:
If the `productDraft.productType.id` variable contains productType name we should rename `productTypeIdToTypeObj` to something like `productTypeNameToTypeObj`.

Reply:
https://github.com/commercetools/commercetools-node-variant-reassignment/pull/57/commits/51c26c3a1605a98f7959435daec1a36e470e745d